openapi.yaml

```diff
@@ -237,6 +237,14 @@ components:
             - KAN_IKKE_HA_TILSVARENDE_BEGRUNNELSE
             - PERSONLOFTER_VALG_HAR_BEHOV_FOR_SEIL_ELLER_SELE
             - ANNEN_KOMMENTAR
+            - EL_RULLESTOL_OPPBEVARING_OG_LAGRING
+            - EL_RULLESTOL_OPPBEVARING_OG_LAGRING_FRITEKST
+            - EL_RULLESTOL_KJENT_MED_FORSIKRING
+            - EL_RULLESTOL_IKKE_KJENT_MED_FORSIKRING
+            - EL_RULLESTOL_HAR_SPESIALSYKKEL
+            - EL_RULLESTOL_HAR_IKKE_SPESIALSYKKEL
+            - EL_RULLESTOL_HENDEL_PLASSERING_HØYRE
+            - EL_RULLESTOL_HENDELPLASSERING_VENSTRE
```

Comment:
På alle disse kan verdien i søknaden være enten true eller false (eller en av to enumverdier på plassering)

Reply:
Trur det er greit å gjere sånn som du har gjort her. Er samme måten vi har gjort det på allerede (feks med `SITTEPUTEVALG_HAR_FRA_FOR` og `SITTEPUTEVALG_TRENGER_SITTEPUTE`). Og datamodellen er ikkje tatt skikkelig i bruk enda, så det gjer ikkje noko om vi ombestemmer oss.

Men bruk enten `..HENDEL_PLASSERING..` eller `..HENDELPLASSERING..` på begge :)